tests/test_provision.py

```diff
@@ -229,3 +229,10 @@ def test_provision_conf_file(tox_project: ToxProjectCreator, tmp_path: Path, rel
     conf_path = str(Path(project.path.name) / "tox.ini") if relative_path else str(project.path / "tox.ini")
     result = project.run("c", "--conf", conf_path, "-e", "py", from_cwd=tmp_path)
     result.assert_success()
+
+
+def test_provision_default_arguments_exists(tox_project: ToxProjectCreator) -> None:
+    ini = "[tox]\nrequires = tox<4"
+    outcome = tox_project({"tox.ini": ini}).run()
```

Comment:
I think you want to do here run `depends`, `list` and not default run.

Reply:
I guess I can as well run all the subcommands as well and not just only `depends` and `list`?

Reply:
Yes the test can be parametrized.